src/conversion/mapper.js

```diff
@@ -159,13 +159,12 @@ export default class Mapper {
 	toModelPosition( viewPosition ) {
 		const data = {
 			viewPosition: viewPosition,
-			modelPosition: this._defaultToModelPosition( viewPosition ),
 			mapper: this
 		};
 
 		this.fire( 'viewToModelPosition', data );
 
-		return data.modelPosition;
+		return data.modelPosition ? data.modelPosition : this._defaultToModelPosition( viewPosition );
```

Comment:
In fact I was curious how such custom mapping may look like, how to create custom mapping without the result of the default mapping, and I realised there is no single test for this change.